plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -820,16 +747,20 @@ def _coast_to_grid(self) -> None:
             np.min(np.linalg.norm(arr - self.source, axis=3)) for arr in self.grids_arr
         )
 
+        tracked_mask = self._tracked_particle_mask()
+
         # Find speed of each particle towards the grid.
-        vmax = np.dot(self.v, self.src_n)
+        vmax = np.dot(self.v[tracked_mask], self.src_n)
 
         # Time for each particle to reach the grid
         t = dist / vmax
 
         # Coast the particles to the advanced position
-        self.x = self.x + self.v * t[:, np.newaxis]
+        self.x[tracked_mask] = (
+            self.x[tracked_mask] + self.v[tracked_mask] * t[:, np.newaxis]
+        )
 
-    def _coast_to_plane(self, center, hdir, vdir, x=None):
+    def _coast_to_plane(self, center, hdir, vdir, x=None, mask=None):
```

Comment:
Can you add parameter descriptions to this docstring? Especially for mask, to just to make that clear